google/cloud/spanner_v1/pool.py

```diff
@@ -316,6 +481,27 @@ def get(self):
             if not session.exists():
                 session = self._new_session()
                 session.create()
+
+        # Set session properties.
```

Comment:
Can we add this is another function altogether? Looks repetitive

Reply:
Added